docs/source/DeveloperDocumentation/PlayerIntegrations/agnoplay.md

```diff
@@ -0,0 +1,92 @@
+# Agnoplay
+
+[Agnoplay](https://www.agnoplay.com/) is a fully agnostic, cloudbased player service.
+
+The [agnoplay](https://github.com/Vualto/agnoplay) repository demonstrates at a lower level how to integrate [VUDRM](https://docs.vualto.com/projects/vudrm/en/latest/index.html) with Agnoplay.
+
+## Example of Agnoplay Integration
+
+This files below can be found in the repo mentioned above here: 
```

Comment:
Typo (this files should be the files)

Reply:
The sentence before made my brain melt, changed!